app/src/main/java/com/github/se/cyrcle/MainActivity.kt

```diff
@@ -36,10 +39,15 @@ class MainActivity : ComponentActivity() {
 
   @Inject lateinit var addressRepository: AddressRepository
 
+  @Inject lateinit var permissionsHandler: PermissionHandlerInterface
+
   override fun onCreate(savedInstanceState: Bundle?) {
     super.onCreate(savedInstanceState)
     requestedOrientation = ActivityInfo.SCREEN_ORIENTATION_PORTRAIT
 
+    permissionsHandler.initHandler(this)
+    Log.e("MainActivity", "PermissionsHandler initialized")
```

Comment:
I don't think logging this as an error (`.e`) is what is intended here.

Reply:
Yep i will remove it thanks

Reply:
Done !